components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -700,12 +700,17 @@ export default class PluginManagement extends AdminSettings<Props, State> {
     handleRemove = async () => {
         this.setState({lastMessage: null, serverError: null});
         if (this.state.removing !== null) {
-            const {error} = await this.props.actions.removePlugin(this.state.removing);
-            this.setState({removing: null});
+            const {disableError} = await this.props.actions.disablePlugin(this.state.removing)
```

Comment:
There is a small linter issue:
```suggestion
            const {disableError} = await this.props.actions.disablePlugin(this.state.removing);
```